cypress/support/step_definitions/event.js

```diff
@@ -6,6 +6,11 @@ When('I click on {string}', (templateSelector) => {
   cy.get(selector).click();
 });
 
+When('I force click on {string}', (templateSelector) => {
+  const selector = nunjucks.renderString(templateSelector, cy.context);
+  cy.get(selector).click({ force: true });
```

Comment:
why don't you always force click instead of recreate step ?

Reply:
done